app/server/utils/ussd/token_processor.py

```diff
@@ -19,16 +23,16 @@ def send_sms(user, message_key, **kwargs):
     @staticmethod
     def send_success_sms(message_key: str, amount: float, user: User, other_user: User, reason: str, tx_time: datetime,
                          balance: float):
-        # TODO: convert amount?
-        TokenProcessor.send_sms(message_key, amount=amount, token_name=default_token(user),
+        # TODO: convert amounts - truncate? need to turn from cents to dollar?
+        TokenProcessor.send_sms(user, message_key, amount=amount, token_name=default_token(user),
                                 other_user=other_user.user_details(), date=tx_time.strftime('%d/%m/%Y'), reason=reason,
                                 time=tx_time.strftime('%I:%M %p'), balance=balance)
 
     @staticmethod
     def exchange_success_sms(message_key: str, user: User, other_user: User, own_amount: float, other_amount: float,
                              tx_time: datetime, balance: float):
-        # TODO: convert amounts?
-        TokenProcessor.send_sms(message_key, own_amount=own_amount, other_amount=other_amount,
+        # TODO: convert amounts - truncate? need to turn from cents to dollar?
```

Comment:
Yeah - might be a 'not now' thing, but different currencies truncate in different ways (some don't show decimal places, some do). This used to be a setting a system level, but probably makes more sense at a token level now.